projects/js-packages/components/components/record-meter-donut/index.tsx

```diff
@@ -0,0 +1,104 @@
+import { __ } from '@wordpress/i18n';
+import React from 'react';
+import './style.scss';
+export type RecordMeterDonutProps = {
```

Comment:
Could we name it something more general, because the donut meter will also be used for as search request meter. And in the future, possibly sharing meter etc 😄 

This also applies to some of the naming of CSS classes etc.

Reply:
sure - that's totally doable. I just named it after its sibling RecordMeterBar component. What about DonutMeter or DonutChart?

Reply:
Okay, gone with DonutMeter for now, seems a bit more general use to me :)